plugins/Login/stylesheets/login.less

```
@@ -65,6 +65,14 @@
 
```

While testing locally I've seen that the login form currently always consumes 33% of the screen width. For a very very wide screen that imho now looks ugly, as the button stretches over the whole width.
I therefor would recommend to limit the max width of the form to a reasonable size, like e.g. 750px

```suggestion

    .contentForm {
        max-width: 750px;
    }

```

---

:+1: